nvflare/app_common/handlers/event_recorder.py

```diff
@@ -123,9 +123,9 @@ def event_error_tag(self, fl_ctx: FLContext):
 
     def validate_prop(self, prop_name: str, req: _CtxPropReq, fl_ctx: FLContext):
         stats = fl_ctx.get_prop(self._KEY_EVENT_STATS, None)
-        assert isinstance(stats, _EventStats)
+        assert isinstance(stats, _EventStats), "stats must be _EventStats but got {}".format(type(stats))
 
-        assert isinstance(req, _CtxPropReq)
+        assert isinstance(req, _CtxPropReq), "req must be _CtxPropReq but got {}".format(type(req))
```

Comment:
these are private classes, who set the stats? req?
do we really need to check the class here?